adapters/handlers/graphql/local/common_filters/near_filters.go

```diff
@@ -112,22 +112,42 @@ var vectorPerTarget = graphql.NewScalar(graphql.ScalarConfig{
 	ParseLiteral: func(valueAST ast.Value) interface{} {
 		switch v := valueAST.(type) {
 		case *ast.ObjectValue:
-			result := make(map[string][]float32)
+			result := make(map[string]interface{})
 			for _, field := range v.Fields {
 				key := field.Name.Value
 				switch value := field.Value.(type) {
 				case *ast.ListValue:
-					floatValues := make([]float32, len(value.Values))
-					for i, value := range value.Values {
-						floatValue, err := strconv.ParseFloat(value.GetValue().(string), 64)
-						if err != nil {
-							return nil
+					if len(value.Values) > 0 {
+						switch value.Values[0].(type) {
+						case *ast.ListValue:
+							// Handle list of lists of floats
+							listOfLists := make([][]float32, len(value.Values))
+							for i, listValue := range value.Values {
+								innerList := listValue.(*ast.ListValue)
```

Comment:
Is it possible that the user may supply a mixed list here such that `value.Values[0]` is `*ast.ListValue` but other entries of `value.Values` are a different type? `L126` would panic in such a case, right? Would it be useful to check here for that?

Reply:
In `adapters/handlers/graphql/local/get/get_test.go` there is a test for that:

```
		query := `{ Get { SomeThing(
						nearVector: {
						vectorPerTarget: {title1: [[1, 0], [0,1]], title2: [0, 0, 1], title3: [0, 0, 0, 1]}
						targets: {
							targetVectors: ["title1", "title1", "title2", "title3"], 
							combinationMethod: manualWeights,
							weights: {title1: [1, 2], title2: 3, title3: 4}
						}
					}) { intField } } }`
```

which works

Reply:
Ah, I meant something broken like:
```graphql
vectorPerTarget: {title1: [[1, "fish"], [0, 1]]}
```
since it's being defined as a scalar type so won't have the same implicit input validation that other types have

Since that input is completely broken, do you think a panic is tolerable or would it be beneficial to have that user error caught and thrown appropriately?

Reply:
Added a test :)

But as I expected that is already cought by the GQL layer and the query does not reach this code